staging/src/k8s.io/kubectl/pkg/cmd/auth/whoami.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	authenticationv1alpha1 "k8s.io/api/authentication/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/printers"
+	authenticationv1alpha1client "k8s.io/client-go/kubernetes/typed/authentication/v1alpha1"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/scheme"
+	"k8s.io/kubectl/pkg/util/templates"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/yaml"
+)
+
+// WhoAmIOptions is the start of the data required to perform the operation. As new fields are added, add them here instead of
+// referencing the cmd.Flags()
+type WhoAmIOptions struct {
+	AuthClient authenticationv1alpha1client.AuthenticationV1alpha1Interface
+
+	TablePrinter printers.ResourcePrinter
+	OutputFormat *string
+
+	genericclioptions.IOStreams
+	PrintFlags *genericclioptions.PrintFlags
+}
+
+var (
+	whoAmILong = templates.LongDesc(`
+		Check who you are and your attributes (groups, extra).
```

Comment:
This should have explicit information that this command is ALPHA.

Reply:
Added the `Experimental` prefix (the same as the `events` command has).